packages/integration-tests/test/actions.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { Config } from '../../../common'
+import { Web3Provider } from '@ethersproject/providers'
+import { ganache } from '@eth-optimism/ovm-toolchain'
+import assert = require('assert')
+
+const {
+  Contract,
+  ContractFactory,
+  providers: { JsonRpcProvider },
+  Wallet,
+} = require('ethers');
+const fs = require('fs')
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
+
+let SimpleStorage
+let L2Messenger
+const L1_USER_PRIVATE_KEY = process.env.L1_USER_PRIVATE_KEY
```

Comment:
These env vars should be pulled from `Config` to follow the same pattern and make it easy to know what env vars are used in the future

Reply:
Good call. I'll move these into the Config object